master/internal/rm/kubernetesrm/pods.go

```diff
@@ -217,8 +222,16 @@ func (p *pods) Receive(ctx *actor.Context) error {
 			}
 		}
 
-		p.startPodInformer(ctx)
-		p.startNodeInformer(ctx)
+		err := p.startPodInformer(ctx)
+		if err != nil {
+			return err
+		}
+
+		err = p.startNodeInformer()
```

Comment:
Our current pattern so far would be to return an `error` as part of `newNodeInformer`, as that is the easiest way an error can be bubbled up, and then we would need to `go p.startNodeInformer()`.  As currently implemented it looks like the call would block here.

Reply:
what error would we return in `NewNodeInformer`? I'm not sure I understand why/how we would want the error to bubble up through there vs in `startInformer` where we execute the functions that actually return those errors.